src/coreclr/build-test.cmd

```diff
@@ -646,6 +642,7 @@ if defined __CompositeBuildMode (
     mkdir !__CompositeOutputDir!
     echo --composite>>!__CompositeResponseFile!
     echo -O>>!__CompositeResponseFile!
+    echo --targetarch:%__BuildArch%>>!__CompositeResponseFile!
```

Comment:
What happens if `!__CompositeOutputDir!` already exists?  Will we append to the already existing `!__CompositeResponseFile!` file?  We might want to `rmdir /s /q !__CompositeOutputDir!` first.

Also I noticed we are using `composite.out` directory name on Windows, but not on Linux.  Could we behave the same way on both OSes?

Reply:
For now I have added a command to delete the response file upfront. For other changes related to framework crossgenning, I would prefer to leave them for a separate PR if you don't mind as I already have a half-working private change switching over framework crossgenning to use R2RTest which would substantially simplify both versions of the build-test script and make many of these concerns moot.